### PR TITLE
Update Form after AWS Feedback

### PR DESCRIPTION
**Pull request by ericboucher**
Quick updates on the form, could you just check that it should work as expected @bencpeters ? :)
You may need to add a column "What type of volunteering are you interested in?" in the dev table.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -374,6 +387,16 @@ <h3>Volunteering Interest</h3>
           ],
         });
 
+        $('select[name="types[]"]').select2({
+          tags: true,
```
Comment (bencpeters) [inline index.html]: Do we want this to be strict set, or an open-ended input? If we only want these 4 options provided, we can just remove this `tags: true` from here (and not worry about validation on the proxy server side). The backend validation and `tags` argument are for multi select fields that we also want people to optionally type their own response.

File: airtable_helpers.py [hunk 2]
```diff
@@ -192,6 +199,8 @@ def _filter_none(values):
     "Are you interested in mentoring?": _checkbox_to_bool("mentor"),
     "What role would you be most interested in playing?":
         lambda d: _filter_in_list(_get_list("roles[]")(d), VALID_ROLES),
+    "What type of volunteering are you interested in?":
+        lambda d: _filter_in_list(_get_list("types[]")(d), VALID_TYPES),
```
Comment (bencpeters) [inline airtable_helpers.py:203]: On the AirTable side, do we want this to be a "multiple select" type (as `roles` is), or a text input that gets comma delimited lists? The disadvantage to the multi select is that the Air Table API doesn't support adding allowed options to that, so we basically have to do the "other_types" paradigm that we did for roles on the Air Table side.

**Review by bencpeters — COMMENTED** [review]
This basically looks good, we just have to decide what kind of field the new "types" field wants to be in 
Air Table, and whether we want the UX to support custom inputs before merging.

**Comment by ericboucher** [comment]
Yes we only want these 4 options for now

Eric Boucher

**Comment by bencpeters** [comment]
Ok! I'll add it to the backend as a multiple select then. In theory, we don't need the `_filter_in_list` call on the proxy server in that case, but in practice it's a good belt & suspenders check, since the whole API call actually fails if it encounters any unexpected values for a field of that type.

**Comment by ericboucher** [comment]
Sounds good!

Eric Boucher

**Pull request merged**